src/widgets/speakers/ui/speakers.test.tsx

```diff
@@ -20,7 +20,7 @@ describe('Speakers', () => {
   });
 
   it('renders the name correctly', () => {
-    const nameElement = screen.getByTestId('contact-name');
+    const nameElement = screen.getByTestId('subtitle');
```

Comment:
Maybe merge the tests here too under the name 'render content correctly'?

Reply:
done